dependencies.json

```diff
@@ -1,13 +1,23 @@
 {
     "*": {
-        ">=3124": [
+        "3124 - 4069": [
```

Comment:
Hmm, I'm unsure if this is the right syntax.

Reply:
Seems valid https://github.com/wbond/package_control/blob/67ef2e02fef941c01694a52c5dfc3a08f86fbe49/example-dependencies.json#L20-L21

Reply:
I don't see the point of those version dependant checks anyway as this code base won't work with ST3 or even older ST4 builds.

Reply:
Fair point, I've removed the checks.